ambari-server/src/main/resources/common-services/HBASE/0.96.0.2.0/widgets.json

```diff
@@ -37,8 +37,8 @@
               "component_name": "HBASE_REGIONSERVER"
```

Comment:
I think this file is not related. It's ancient `common-services/HBASE/0.96.0.2.0` that is shared between a few stacks. The actual metadata is inherited from `stacks/HDP/2.3/services/HBASE`